src/main/java/org/betonquest/betonquest/compatibility/protocollib/ProtocolLibIntegrator.java

```diff
@@ -57,6 +59,7 @@ public void reload() {
         if (MythicHider.getInstance() != null) {
             MythicHider.start();
         }
+        npcGlow.stop();
```

Comment:
why is there only a stop when a reload is executed?

Reply:
Oh sorry I should name it `refresh` instead stop. What it does it make all in NPCGlow fresh (clear all map) and load it back without making new instance.